pom.xml

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.exasol</groupId>
+	<artifactId>postgresql-virtual-schema</artifactId>
+	<version>0.1.0</version>
+	<name>Virtual Schema for Postgresql</name>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<java.version>11</java.version>
+		<surefire.and.failsafe.plugin.version>3.0.0-M3</surefire.and.failsafe.plugin.version>
```

Comment:
Check for newer versions. M4 should work as far as I remember, but M5 produces 0% coverage on Sonar cloud. Please compare with other recent projects.

Reply:
M4 does not fail tests if beforeAll fails. So M3 it is